android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -618,6 +618,7 @@ public void receiveCommand(WebView root, int commandId, @Nullable ReadableArray
         if (args == null) {
           throw new RuntimeException("Arguments for loading an url are null!");
         }
+        isOverriddenLoad = false;
```

Comment:
Perhaps this should be called `isWaitingForCommandLoadUrl`. If you look at the lifecycle of the two places this is mutated, what the variable seems to really be doing is signifying that we are waiting for `onShouldStartLoadWithRequest` to complete and to give us the COMMAND_LOAD_URL so we can proceed with loading the url.

Reply:
Changed to `waitingForCommandLoadUrl`.